src/store/reducers/userReducers.js

```diff
@@ -0,0 +1,28 @@
+import InitialState from "../initialState";
```

Comment:
Rename your reducer to `user.reducer.js`
Same convention for all reducers too.. thanks